docs/AboutUs.md

```diff
@@ -4,7 +4,7 @@ Display |         Name         |             Github Profile             | Portfo
 --------|:--------------------:|:--------------------------------------:|:---------:
 ![](team/chuahanyongdarren.png) | Chua Han Yong Darren | [Github](https://github.com/chydarren) | [Portfolio](team/chuahanyongdarren.md)
 ![](team/chiathinhong.png) |    Chia Thin Hong    |   [Github](https://github.com/wcwy)    | [Portfolio](team/chiathinhong.md)
-![](https://via.placeholder.com/100.png?text=Photo) |    Yong Chin Han     | [Github](https://github.com/chinhan99) | [Portfolio](team/yongchinhan.md)
+![](team/yongchinhan.jpg) |    Yong Chin Han     | [Github](https://github.com/chinhan99) | [Portfolio](team/yongchinhan.md)
```

Comment:
Nice profile image! The aspect ratio might wish to update it to 150 x 150?

Reply:
Ammended! Thanks!